src/event/handlers/CommandParserHandler.ts

```diff
@@ -31,6 +32,10 @@ class CommandParserHandler extends EventHandler {
 			if (this.commandFactory.commandExists(trigger)) {
 				const command = this.commandFactory.getCommand(trigger);
 
+				if (command instanceof CommandListCommand) {
+					command.setCommands(this.commandFactory.getCommandsWithoutAliases());
+				}
```

Comment:
I know performance isn't exactly an issue, but isn't that going to run everytime `?commands` is triggered? I mean, we could store the list as a static property (set in `CommandParserHandler` constructor) rather than have to query it everytime.

Reply:
I think it's fine for now honestly, I mean we create a new `CommandFactory` every time a message is sent... 😬